spring-kafka/src/test/java/org/springframework/kafka/annotation/EnableKafkaIntegrationTests.java

```diff
@@ -82,6 +82,7 @@
 import org.springframework.kafka.support.converter.StringJsonMessageConverter;
 import org.springframework.kafka.test.rule.KafkaEmbedded;
 import org.springframework.kafka.test.utils.KafkaTestUtils;
+import org.springframework.lang.NonNull;
```

Comment:
This cannot be backported to `1.3.x` because it was introduced since Spring `5.0`.

However I'll replace it with some other Spring `4.3` compatible parameter level annotation during backport